main.py

```diff
@@ -0,0 +1,266 @@
+import repetitive
+
+# Individual values of members to be used for comparisons
+elizabeth = 1
+marj = 2
+rosanne = 3
+ruthanne = 4
+donnie = 5
+patricia = 6
+kim = 7
+monica = 8
+martin = 9
+gabe = 10
+john = 11
+ben = 12
+jerome = 13
+johnG = 14
+jamie = 15
+julia = 16
+phil = 17
+alley = 18
+anna = 19
+steven = 20
+ellane = 21
+jake = 22
+veera = 23
+precila = 24
+clarence = 25
+lincoln = 26
+mo = 27
+alex = 28
+
+# The list of members organized into groups based on generation
+
+gen1 = [elizabeth] # 1
+
+gen2 = [marj, rosanne, ruthanne, donnie, patricia] # 2, 3, 4, 5, 6
+
+gen3 = [kim, monica, martin, gabe, john, ben, jerome] # 7, 8, 9, 10, 11, 12, 13
+
+gen4 = [johnG, jamie, julia, phil, alley, anna, steven, ellane, jake, veera, precila, clarence] # 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25
+
+gen5 = [lincoln, mo] # 26, 27
+
+gen6 = [alex] # 28
+
+# List of members who don't have children of their own
+# The members in this list cannot be classified as parents, grandparents, etc.
+hasNoChildren = [rosanne, ruthanne, donnie, patricia, martin, john, ben, jerome, johnG, jamie, julia, alley, anna, steven, jake, veera, precila, clarence, mo, alex]
+
+# while loop
+while(1):
+
+  repetitive.list()
+  
+  # User inputs to be stored as a value
+  userInput = int(input())
+  userInput2 = int(input())
+  
+  if userInput == userInput2:
+    print('You selected the same member twice! Please select 2 different members.')
+  
+  if userInput in gen1 and userInput != userInput2:
+    print("Elizabeth")
+  
+  if userInput in gen2 and userInput != userInput2:
+    if userInput == marj:
```

Comment:
The setup of the data can be tailored to a specific family tree, but your algorithm has to be generalized to any family tree.  So it cannot refer directly to any specific family member.... you don't know that in advance.